vevestaX/vevesta.py

```diff
@@ -306,11 +307,23 @@ def __profilingReport(self, fileName):
 
         profileOfVariableDataframe = pandas.DataFrame(
             {"Field Name": ["Distinct", "Distinct (%)", "Missing", "Missing (%)", "Infinite", "Infinite (%)", "Mean",
-                            "Minimum", "Maximum", "Zeros", "Zeros (%)", "Negative", "Negative (%)",
-                            "Total Memory Size(bytes)"]})
+                            "Minimum", "Maximum", "Zeros", "Zeros (%)", "Negative", "Negative (%)", "Kurtosis",
+                            "Skewness", "Median", "Mode", "Outliers", "Outliers (%)", "Q1 quantile", "Q2 quantile",
+                            "Q3 quantile", "100th quantile", "Total Memory Size(bytes)"]})
 
         numericColumns = self.__data.select_dtypes(include=["number"]).columns
         for col in numericColumns:
+            # finding outliers for each column
+            Q1 = np.quantile(self.__data[col], 0.25)
+            Q3 = np.quantile(self.__data[col], 0.75)
+            IQR = Q3 - Q1
+            low_lim = Q1 - 1.5 * IQR
+            up_lim = Q3 + 1.5 * IQR
+            outliers = []
+            for x in self.__data[col]:
```

Comment:
There is a way to calculate it in pandas, please use that...don't write for loops